src/packages/cli/src/bin.ts

```diff
@@ -174,9 +177,14 @@ async function main(): Promise<number> {
     return 1
   }
   console.log(result)
+
+  // Project hash is a SHA256 of the schemaPath
+  const projectHash: string = await getProjectHash()
+
   // check prisma for updates
   const checkResult = await checkpoint.check({
     product: 'prisma',
+    project: projectHash,
```

Comment:
Just to clarify, a `project` is an app, right? And you can have many projects that use the Prisma CLI `product`?

Reply:
Exactly